core.py

```diff
@@ -285,7 +288,7 @@ def store_file(fd, filename, md5, arcpath, arcname):
             self.abort(404, 'no such ticket')
         arcpath = os.path.join(self.app.config['upload_path'], ticket_id + '.tar')
 
-        if self.request.get('complete').lower() not in ['1', 'true']:
+        if self.request.GET.get('complete', '').lower() not in ('1', 'true'):
```

Comment:
I don't understand this change - is it logically identical? Was it broken before?